docs/ko/week02/02-1.md

```diff
@@ -0,0 +1,370 @@
+
+
+| <center><img src="{{site.baseurl}}/images/week02/02-1/Figure1.jpg" alt="Figure1" style="zoom: 33%;" /></center> |
+<!-- | <center>Figure 1: Computation Graph representation for a Parametrised Model </center>| -->
+| <center>그림 1: 매개변수화된 모델에 대한 연산 그래프 </center>|
+
+<!-- Examples of parametrised functions - -->
+매개변수화된 함수의 예시
+<!-- - Linear Model - Weighted Sum of Components of the Input Vector : -->
+- 선형 모델 - 입력 벡터의 각 요소들에 대한 가중합:
+  $$
+  \bar{y} = \sum_i w_i x_i \text{  ;  } C(y,\bar{y}) = ||y - \bar{y}||^2
+  $$
+
+<!-- - Nearest Neighbor - There is an input x and a weight matrix W with each row of the matrix indexed by k. The output is the value of k that corresponds to the row of W that is closest to x : -->
+- 최근접 이웃 - 입력 x와 k로 인덱싱할 수 있는 행을 가진 가중치 행렬 W가 있습니다. 이때의 출력은 k의 값 자체가 될텐데, 이 값은 x와 가장 가까운 W의 행을 나타냅니다.:
+  $$
+  \bar{y} = \text{argmin}_k ||x - w_{k,.} ||^2
+  $$
+  <!-- Parameterized models could also involve complicated functions. -->
+  매개변수화된 모델은 복잡한 함수와도 관련있을 수 있습니다.
+
+
+<!-- #### Block diagram notations for computation graphs -->
+#### 연산 그래프에 대한 각 요소에 대한 표현
+
+<!-- - Variables (tensor, scalar, continuous, discrete)
+    - <img src="{{site.baseurl}}/images/week02/02-1/x.PNG" alt="x" style="zoom:50%;" /> is an observed input to the system
+    - <img src="{{site.baseurl}}/images/week02/02-1/y.PNG" alt="y" style="zoom:50%;" /> is a computed variable which is produced by a deterministic function -->
+
+- 변수 (텐서, 스칼라, 연속적인 값, 비연속적인 값)
+    - <img src="{{site.baseurl}}/images/week02/02-1/x.PNG" alt="x" style="zoom:50%;" /> 는 시스템상에서 관찰된 입력값입니다.
+    - <img src="{{site.baseurl}}/images/week02/02-1/y.PNG" alt="y" style="zoom:50%;" /> 는 결정적 함수에 의해서 생성된 계산된 값입니다.
+
+
+<!-- - Deterministic functions
+
+    <img src="{{site.baseurl}}/images/week02/02-1/deterministic_function.PNG" alt="deterministic_function" style="zoom:50%;" />
+
+    - Takes in multiple inputs and can produce multiple outputs
+    - It has an implicit parameter variable (${w}$)
+    - The rounded side indicates the direction in which it is easy to compute. In the above diagram, it is easier to compute ${\bar{y}}$ from ${x}$ than the other way around -->
+
+- 결정적 함수
+
+    <img src="{{site.baseurl}}/images/week02/02-1/deterministic_function.PNG" alt="deterministic_function" style="zoom:50%;" />
+
+    - 여러 개의 입력을 받고 여러 개의 출력을 생성합니다.
+    - 내부적으로 함축된 매개변수 (${w}$) 를 가지고 있습니다.
+    - 곡선으로 나타난 부분은 어느 쪽이 계산하기 쉬운지를 방향으로 표현한 것입니다. 위의 다이어그램에서는 ${x}$ 으로부터 ${\bar{y}}$ 를 계산하는 것이 다른 방법보다 쉽습니다.
+
+
+
+- Scalar-valued function
+
+  <img src="{{site.baseurl}}/images/week02/02-1/scalar-valued.PNG" alt="scalar-valued" style="zoom:50%;" />
+
+    - Used to represent cost functions
+    - Has an implicit scalar output
+    - Takes multiple inputs and outputs a single value (usually the distance between the inputs)
```

Comment:
There's a lot of English, still.

Reply:
A ha. I got it.
It was just 02.md for this pr. And he is working on 02-1 now.

Reply:
Ah, Sorry for confusing. I should work it locally.